lib/webauthn/configuration.rb

```diff
@@ -13,6 +14,8 @@ def self.configure
   end
 
   class Configuration
+    class RootCertificateFinderNotSupportedError < Error; end
```

Comment:
I know we haven't been 100% consistent about this, but I think going forward I would prefer we have errors namespaced with `WebAuthn` module only, so users don't get a very long error class reference when getting the exception.

I would consider removing namespaces in the future from the ones that have more that one nesting.

Reply:
Thanks!